traffic_ops/traffic_ops_golang/server/update.go

```diff
@@ -155,83 +153,55 @@ func parseQueryParams(params map[string]string) (*updateValues, error) {
 	// Verify query string parameters
 	configUpdatedBoolParam, hasConfigUpdatedBoolParam := params["updated"]     // Deprecated, but still required for backwards compatibility
 	revalUpdatedBoolParam, hasRevalUpdatedBoolParam := params["reval_updated"] // Deprecated, but still required for backwards compatibility
-	configUpdateTimeParam, hasConfigUpdateTimeParam := params["config_update_time"]
-	revalidateUpdateTimeParam, hasRevalidateUpdateTimeParam := params["revalidate_update_time"]
 	configApplyTimeParam, hasConfigApplyTimeParam := params["config_apply_time"]
 	revalidateApplyTimeParam, hasRevalidateApplyTimeParam := params["revalidate_apply_time"]
 
 	if !hasConfigApplyTimeParam && !hasRevalidateApplyTimeParam &&
-		!hasConfigUpdateTimeParam && !hasRevalidateUpdateTimeParam &&
 		!hasConfigUpdatedBoolParam && !hasRevalUpdatedBoolParam {
-		return nil, errors.New("must pass at least one query parameter: 'config_apply_time', 'revalidate_apply_time', 'config_update_time', 'revalidate_update_time' (may also pass bool `update` `reval_updated`)")
+		return nil, errors.New("must pass at least one of the following query parameters: 'config_apply_time', 'revalidate_apply_time' ,'update', 'reval_updated'")
```

Comment:
only the smallest comment. this should have `updated` instead of `update`

Reply:
DOH. Fixed.